openforcefield/tests/test_topology.py

```diff
@@ -188,6 +190,21 @@ def test_box_vectors(self):
             topology.box_vectors = good_vectors
             assert (topology.box_vectors == good_vectors * np.eye(3)).all()
 
+    def test_is_periodic(self):
+        """Test the getter and setter for is_periodic"""
+        vacuum_top = Topology()
+        assert vacuum_top.is_periodic is False
```

Comment:
I probably go back and forth - the only thing I'll consciously do is avoid the `if not:` conditional since I've been burned on falsiness many times.

I looked it up, and if the object is reliably a boolean, there is no difference, since something that evaluates to `False` will only ever end up at the same place in memory. The first two answers in this thread are probably the best summary of my findings: both are fine, but `is` is marginally better for the built-in boolean types as it avoids `__eq__` business - but the opposite is true for derived types like NumPy data. 

https://stackoverflow.com/questions/27276610/boolean-identity-true-vs-is-true

Reply:
Oh, interesting. Thanks for checking into this! I'll leave it up to implementer's choice then :-)